packages/web-react/src/types/accordion.ts

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ChildrenProps, StyleProps } from './shared';
+
+export type AccordionElementType = 'section' | 'article' | 'div';
```

Comment:
Isn't those two the same type? Can we use just one?

Reply:
Technically it is one type, but here there is a type for two components and each of them has a different default value, so I would prefer to keep this.

Reply:
I think that the default value has no effect on this. Is there any benefit to leaving these two types separate when they are the same?

Reply:
Whatever, I just gave my insignificant opinion on it.